pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerBkEnsemblesTests.java

```diff
@@ -340,5 +340,32 @@ public void testSkipCorruptDataLedger() throws Exception {
         client.close();
     }
 
+    @Test
+    public void testTopicWithWildCardChar() throws Exception {
+        PulsarClient client = PulsarClient.builder().serviceUrl(adminUrl.toString()).statsInterval(0, TimeUnit.SECONDS)
+                .build();
+
+        final String ns1 = "prop/usc/topicWithSpecialChar";
+        try {
```

Comment:
Seems this is not needed?

Reply:
yes, it doesn't need but in one of the previous failure log, I saw error-message that "namespace already exist" so, I just added to fix it. However, issue seems something else where threads are stuck because bookkeeper ensemble is not up. and it's kind of weird because this test works fine locally. will look into it.